File: 연습문제/백준2480.js
const numbers = require('fs').readFileSync('예제.txt').toString().split(' ').map(Number);

const a = numbers[0];
const b = numbers[1];
const c = numbers[2];

let money = 0;
let maxNum = 0;

if(a === b && a === c){
  money = 10000 + a*1000;
} else if(a === b || a === c || b === c){
  if(a === b || a === c){
    money = 1000 + a*100;
  } else{
    money = 1000 + b*100;
  }
} else{
   maxNum = Math.max(...numbers);
   money = maxNum*100;
}

console.log(money);

File: 연습문제/백준2480.py
a, b, c = map(int, input().split())

if a == b == c:
    print(10000+a*1000)
elif a == b:
    print(1000+a*100)
elif a == c:
    print(1000+a*100)
elif b == c:
    print(1000+b*100)
else:
    print(100 * max(a,b,c))

    
--------------------------------------------js

const input = require("fs")
    .readFileSync("/dev/stdin")
    .toString()
    .split(" ")
    .map((value) => +value);

input.sort(function (a, b) {
    return b - a;
});

const [A, B, C] = input;

if (A !== B && A !== C && B !== C) {
    console.log(A * 100);
} else if (A === B && B !== C) {
    console.log(1000 + A * 100);
} else if (B === C && C !== A) {
    console.log(1000 + C * 100);
} else if (C === A && A !== B) {
    console.log(1000 + C * 100);
} else {
    console.log(10000 + A * 1000);
}
